src/net/Api/Controllers/ConfigController.cs

```diff
@@ -29,5 +29,12 @@ public async Task<ConfigurationSetting> Get([FromQuery] string name)
             //TODO: Move try/catch to here?
             return await clients.ConfigurationClient.GetConfigurationSettingAsync(name);
         }
+
+        [HttpGet]
+        [Route("/config/searchTerm")]
+        public string GetSearchTerm()

```

Comment:
Let's talk about how we want to deal with search term.  At this point, I don't want to add the ability to free form search and the POST with URL will go behind auth so all those images are tied to an account and only visible to that account by default.

Reply:
took that out entirely. totally agree.